www/source/partials/docs/_bp-kubernetes.html.md.erb

```diff
@@ -25,7 +33,7 @@ $ kubectl get pods -l run=mytutorial
 
 ## Environment variables and Networking
 
-Kubernetes supports passing [environment variables](http://kubernetes.io/docs/user-guide/environment-guide/) into containers, which can be done via the Habitat Operator.
+Kubernetes supports passing [environment variables](http://kubernetes.io/docs/user-guide/environment-guide/) into containers, which can be done [via the Habitat Operator](https://github.com/kinvolk/habitat-operator/tree/master/examples/env-vars).
 
 ## Multi-container Pods
```

Comment:
> Multi-container pod support through Habitat is still under active development as part of the Habitat Operator.

Really?

Reply:
That was there already.

Reply:
But do we plan on supporting that? Maybe we should create a ticket not to forget?

Reply:
/cc @tashimi